tests/e2e/initDevfileTests.sh

```diff
@@ -23,6 +23,16 @@ launchSingleUserstory(){
 checkUserstoryName(){
     local checkedName="$(ls tests/devfiles/${TS_SELENIUM_EDITOR} | grep ${USERSTORY}.spec.ts)";
 
+    if [ -z "$TS_SELENIUM_EDITOR"]; then
+        echo ""
+        echo "Variable TS_SELENIUM_EDITOR is unset."
+        echo ""
+        echo "Please assign the variable to a correct editor value."
```

Comment:
It would be also helpful to mention currently supported values: `['che-code', 'theia']`
https://github.com/eclipse/che/blob/main/tests/e2e/TestConstants.ts#L21-L22

Reply:
Yes, I thought about it. But I might open another issue that would suggest all valued that are in che/tests/e2e/test/devfiles which are currently `['che-code', 'theia']`. Or should I include it in this PR?

Reply:
IMHO, no need to open a new issue, because showing the options is a part of expected behavior described in target issue [1], if I understood it correctly:
> The error message should tell us that the TS_SELENIUM_EDITOR variable is empty and **correctly show the options**.

[1] https://github.com/eclipse/che/issues/22018

Reply:
Fixed.